h/datasetjson.h

```diff
@@ -62,6 +63,7 @@ void respondWithDataset(HttpResponse* response, char* absolutePath, int jsonMode
 void respondWithVSAMDataset(HttpResponse* response, char* absolutePath, hashtable *acbTable, int jsonMode);
 void respondWithDatasetMetadata(HttpResponse *response);
 void respondWithHLQNames(HttpResponse *response, MetadataQueryCache *metadataQueryCache);
+int qsamAllocFile( qsamFileRequest *request, char* message, int messageLength);
```

Comment:
There are no QSAM files or datasets, QSAM is an access method.

Reply:
From earlier comment.  I am open to any name change from qsam.  Following names in the jira ticket.

Reply:
@ifakhrutdinov is correct. I would name it ```allocDataSet``` which covers both PS and PDS(E) data sets.